pkg/kubelet/kuberuntime/kuberuntime_container.go

```diff
@@ -810,6 +813,7 @@ func (m *kubeGenericRuntimeManager) removeContainer(containerID string) error {
 	if err := m.removeContainerLog(containerID); err != nil {
 		return err
 	}
+	m.cpuManager.UnregisterContainer(containerID)
```

Comment:
what do we do in case we get an error while unregistering the container ?

Reply:
This has been updated so that errors are surfaced to the caller of killContainer and removeContainer.